app/controllers/admin/categories_controller.rb

```diff
@@ -2,6 +2,9 @@ class Admin::CategoriesController < ApplicationController
 
   before_filter :ensure_is_admin
 
+  #Allow admin to access admin panel before email confirmation
+  skip_filter :cannot_access_without_confirmation
```

Comment:
@ovan I decided to first do even less all the way approach and only introduce AdminAreaController. (I chose to add "Area" to the name as AdminController/UserController might sound more like something that handles those directly (compared to PeopleController) but if you think AdminController would be better name, it might be. It was actually the UserController name that I didn't feel fully comfortable with for this purpose. 

Anyway I decided now to leave it out, as I also consider this as a test of if it has any effect to conversion to free admins a bit from the email confirmation. Based on results, we might end up changing this quite soon, so that the limit is no more in the admin panel only (but allow admins to also view the marketplace without logging in. And if that would be done, then there's nothing special in the UserAreaController. So I decided to now take only the first step as the benefit from that feels very concrete and clear.

All admin controllers now inherit AdminAreaContorller, so no need to repeat the filter that ensures Admin rights or the exception of not requiring email confirmation. And I do think here it fits to logic quite well to handle it as exception for the Admins rather than a requirement only for the user area.

Although it is true, that skipping filters can always be a bit of a surprise hidden in one layer of controllers.. :/

So feel free to comment. Anyway I think this is better than repeating the exception everywhere. And this takes a first (smaller) step in this refactoring direction and UserAreaController could be added later if it seems useful. Now it would be just one row of content there.